BGAnimations/ScreenSelectMusic overlay/PaneDisplay.lua

```diff
@@ -106,13 +106,27 @@ local GetScoresRequestProcessor = function(res, master)
 					end
 
 					if gsEntry["isSelf"] then
-						SetNameAndScore(
-							GetMachineTag(gsEntry),
-							string.format("%.2f%%", gsEntry["score"]/100),
-							playerName,
-							playerScore
-						)
-						personalRecordSet = true
+						-- Let's check if the GS high score is higher than the local high score
+						local player = i - 1
```

Comment:
Can you use enum style for. GetSongAndSteps technically wants an enum and not a number (even though it does work). I think `PlayerNumber[i]` should work.

Reply:
Sure, on it!